Sources/UserDetailView.swift

```diff
@@ -0,0 +1,20 @@
+import SwiftUI
+
+struct UserDetailView: View {
+    var owner: Owner
+
+    var body: some View {
+        Text("User Detail view for user: \(owner.login)")
+    }
+}
+
+struct UserDetailView_Previews: PreviewProvider {
+    static var previews: some View {
+        UserDetailView(owner: Owner(
+            id: 0,
+            login: "Kurac",
```

Comment:
Avoid using curse words of any kind, you can never be sure who will continue to work on the repo and might find shit like this. 💩🙈